### PR TITLE
FIDO U2F Support

### PR DESCRIPTION
**Pull request by dschuermann**
As requested in https://github.com/nextcloud/android/pull/4286

### CODE REVIEW

File: app/src/main/java/com/nextcloud/talk/controllers/WebViewLoginController.java [hunk 1]
```diff
@@ -169,6 +175,8 @@ protected void onViewBound(@NonNull View view) {
         webView.clearHistory();
         WebView.clearClientCertPreferences(null);
 
+        webViewFidoBridge = WebViewFidoBridge.createInstanceForWebView((AppCompatActivity) getActivity(), webView);
```
Comment (mario) [inline app/src/main/java/com/nextcloud/talk/controllers/WebViewLoginController.java]: Maybe we should do a check if getActivity() != null?

Comment (dschuermann) [reply app/src/main/java/com/nextcloud/talk/controllers/WebViewLoginController.java]: Hm, if getActivity() is null a lot of other lines would also fail (such as the next line with ``CookieSyncManager.createInstance(getActivity());``).
What about an early exit for the whole ``onViewBound()`` method if getActivity is null?

Comment (mario) [reply app/src/main/java/com/nextcloud/talk/controllers/WebViewLoginController.java]: I don't disagree with you. There's a need to rewrite this whole thing to make it better. So it's probably fine for now :)

**Comment by mario** [comment]
Thanks. One more thing before I can merge it - it looks like your commits are not signed. Could you please do it?

Sorry for the hassle!

**Comment by dschuermann** [comment]
> Thanks. One more thing before I can merge it - it looks like your commits are not signed. Could you please do it?
> 
> Sorry for the hassle!

As the maintainer of OpenKeychain, I concur ;)

**Comment by mario** [comment]
Thank you so much!

**Comment by jancborchardt** [comment]
@dschuermann by the way, we have a "Talk development chat" for all contributors of the app. If you like, we can add you there with a guest account? :)

**Pull request merged**